packages/contracts/test/integration/commitReveal.spec.ts

```diff
@@ -0,0 +1,232 @@
+import * as ethers from "ethers";
+
+import * as Utils from "@counterfactual/test-utils";
+import Multisig from "../helpers/multisig";
+
+const CommitRevealApp = artifacts.require("CommitRevealApp");
+
+const web3 = (global as any).web3;
+const { provider, unlockedAccount } = Utils.setupTestEnv(web3);
+
+const [A, B] = [
+  // 0xaeF082d339D227646DB914f0cA9fF02c8544F30b
+  new ethers.Wallet(
+    "0x3570f77380e22f8dc2274d8fd33e7830cc2d29cf76804e8c21f4f7a6cc571d27",
+    provider
+  ),
+  // 0xb37e49bFC97A948617bF3B63BC6942BB15285715
+  new ethers.Wallet(
+    "0x4ccac8b1e81fb18a98bbaf29b9bfe307885561f71b76bd4680d7aec9d0ddfcfd",
+    provider
+  )
+];
+
+const computeStateHash = (
```

Comment:
I'm not sure how others feel, but I think we should avoid this type of syntax for standalone functions and instead use

`function computeStateHash(...) {...}`, passing in `A` and `B` if you want them.

The arrow syntax is useful for inline anonymous functions w/ binding, but closing over global state like this makes things harder to understand when you read it for the first time.

Reply:
Agreed